debian/patches/ocf-suexec.patch

```diff
@@ -1,23 +1,26 @@
 --- a/support/suexec-ocf.c
 +++ b/support/suexec-ocf.c
-@@ -14,6 +14,7 @@
-  * limitations under the License.
-  */
- 
-+
- /*
-  * suexec.c -- "Wrapper" support program for suEXEC behaviour for Apache
-  *
-@@ -69,6 +70,8 @@
+@@ -69,6 +69,9 @@
  
  #define AP_ENVBUF 256
  
 +#define AP_DOC_ROOT "/services"
++#define AP_SUEXEC_ROOT "/services/http/suexec"
```

Comment:
IMO this shouldn't be on NFS.

These are scripts and should probably be in source control (puppet?).

Reply:
Ah yeah, I'd agree with that, I thought it was on NFS for a reason, but I should have asked about that. Putting this in puppet definitely makes more sense.

Reply:
Alright, I moved this to `/usr/local/bin/suexec` (in its own subdirectory to make sure that other scripts in `/usr/local/bin` aren't possible to run using suexec).